charged-ieee/lib.typ

```diff
@@ -31,20 +31,28 @@
   set document(title: title, author: authors.map(author => author.name))
 
   // Set the body font.
-  set text(font: "TeX Gyre Termes", size: 10pt)
+  // As of 2024-08, the IEEE LaTeX template uses wider interword spacing
+  // - See e.g. the definition \def\@IEEEinterspaceratioM{0.35} in IEEEtran.cls
+  set text(font: "TeX Gyre Termes", size: 10pt, spacing: .35em)
 
   // Enums numbering
   set enum(numbering: "1)a)i)")
 
   // Tables & figures
   set figure(placement: top)
+  show figure.caption: set align(left)
   show figure.where(kind: table): set figure.caption(position: top)
   show figure.where(kind: table): set text(size: 8pt)
-  show figure.caption.where(kind: table): smallcaps
-  show figure.where(kind: table): set figure(numbering: "I")
-
+  show figure.where(kind: table): set figure(supplement: [TABLE], numbering: "I")
   show figure.where(kind: image): set figure(supplement: [Fig.], numbering: "1")
-  show figure.caption: set text(size: 8pt)
+
+  // Caption vs. reference:  supplement "TABLE" -> "Table", "Fig." -> "Figure"
+  set ref(supplement: it => {
+    if it.supplement == [TABLE] [Table]
+    else if it.supplement == [Fig.] [Figure]
```

Comment:
The IEEE transaction howto states:
> When referring to figures in typical IEEE papers, authors
should use the abbreviation “Fig.”, but in IEEE Computer
Society conference papers they should use the full word
“Figure”.

So, should we use _Fig._ or _Figure_?

Reply:
Many users will want to use "Figure" and many will want to use "Fig.", which is why it may be a good idea to make that available as an option.

I implemented that in [686c6f216da7b0bfadd5a0586fc4522b50832ca7](https://github.com/typst/templates/pull/50/commits/686c6f216da7b0bfadd5a0586fc4522b50832ca7)